modelscope/outputs/outputs.py

```diff
@@ -613,6 +613,7 @@ class OutputKeys(object):
     #    "output": "我想吃苹果"
     # }
     Tasks.text_error_correction: [OutputKeys.OUTPUT],
+    Tasks.word_alignment: [OutputKeys.OUTPUT],
```

Comment:
show word_alignment output as comment

Reply:
Thank you for your suggestion, I have updated code and added comment .